doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -297,18 +241,12 @@ Some operations use partial representation of entities:
 
 * `id` and `type` are not allowed in update operations, as they are immutable properties.
 
-* In requests where entity `type` is allowed, it may be omitted. When omitted in entity
-  creation operations, the default string value `none` is used for the type.
-
```

Comment:
It seems so... but not sure yet. It depends on how we solve https://github.com/telefonicaid/fiware-orion/issues/2223.

Being conservative, it is better not saying anything about this in this release candidate (next ones will do).